src/style/color.rs

```diff
@@ -480,6 +480,7 @@ mod tests {
 
     #[cfg(feature = "palette")]
     use palette::{Hsl, Hsluv};
+    #[cfg(feature = "palette")]
```

Comment:
unrelated - but clippy was complaining when the palette feature is not enabled. not worth a separate PR to fix.